### PR TITLE
[WIP] Find qcow2 image for zkvm(svirt) backend

### PR DESCRIPTION
**Pull request by SergioAtSUSE**
Fix from: https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/6191

- Related ticket: https://progress.opensuse.org/issues/36754
- Verification run: http://slindomansilla-vm.qa.suse.de/tests/955

### CODE REVIEW

File: lib/bootloader_setup.pm [hunk 1]
```diff
@@ -767,8 +767,10 @@ sub zkvm_add_disk {
     if (my $hdd = get_var('HDD_1')) {
         my $basename = basename($hdd);
         my $basedir  = svirt_host_basedir();
-        my $hdd_dir  = "$basedir/openqa/share/factory/hdd";
-        chomp(my $hdd_path = `find $hdd_dir -name $basename | head -n1`);
+        my $hdd_share_dir  = "$basedir/openqa/share/factory/hdd";
+        my $hdd_cache_dir  = "$basedir/openqa/share/factory/hdd";
```
Comment (okurz) [inline lib/bootloader_setup.pm]: that's the same path

File: lib/bootloader_setup.pm [hunk 2]
```diff
@@ -765,10 +765,12 @@ sub tianocore_select_bootloader {
...
     my ($svirt) = @_;
     if (my $hdd = get_var('HDD_1')) {
-        my $basename = basename($hdd);
-        my $basedir  = svirt_host_basedir();
-        my $hdd_dir  = "$basedir/openqa/share/factory/hdd";
-        chomp(my $hdd_path = `find $hdd_dir -name $basename | head -n1`);
+        my $basename      = basename($hdd);
+        my $basedir       = svirt_host_basedir();
+        my $hdd_share_dir = "$basedir/openqa/share/factory/hdd";
+        my $hdd_cache_dir = "$basedir/openqa/cache";
```
[the remark this answers is not in the document]
Comment (SergioAtSUSE) [reply lib/bootloader_setup.pm]: No, that was before.
When using **$svirt->get_cmd_output**, it is called on s390p8.suse.de

**Comment by foursixnine** [comment]
@SergioAtSUSE Do you plan to look at this ever again?

**Comment by SergioAtSUSE** [comment]
@foursixnine, hopefully one day this ticket will be planned again in the sprint backlog: https://progress.opensuse.org/issues/44468

@Soulofdestiny ?

**Pull request closed without merging**